Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModule.java

```diff
@@ -94,4 +99,26 @@ public void run() {
         };
         AzureUIRefreshCore.addListener(id, listener);
     }
+
+    @Override
+    public void renderChildren(@NotNull final List<ResourceEx<WebApp>> resourceExes) {
+        for (final ResourceEx<WebApp> resourceEx : resourceExes) {
+            final WebApp app = resourceEx.getResource();
+            final WebAppNode node = new WebAppNode(this,
+                resourceEx.getSubscriptionId(), app.id(), app.name(), app.state(), app.defaultHostName(),
+                new HashMap<String, String>() {
+                    {
+                        put("regionName", app.regionName());
+                    }
+                });
+
+            addChildNode(node);
+
+            try {
```

Comment:
Where comes the exception?

Reply:
Incorrect signature form the `refreshItems()`, will remove it.